zerver/tests/test_audit_log.py

```diff
@@ -1178,7 +1178,7 @@ def test_user_group_creation(self) -> None:
             realm=hamlet.realm,
             event_time__gte=now,
             event_type=RealmAuditLog.USER_GROUP_DIRECT_USER_MEMBERSHIP_ADDED,
-        )
+        ).order_by("id")
```

Comment:
Why is this changed here?

Reply:
I don't understand why this entirely unrelated test was failing due to the changes, but looking at this test, we expect the log entries to be sorted, and without using `order_by`, there's no guarantee that the result would be sorted as we expect. 

I guess this would have come up sooner or later.

Reply:
I have removed it since I am unable to reproduce it. I wonder what caused it in the first place.